Indonesia/indo.md

```diff
@@ -0,0 +1,5 @@
+Petualangan kamu dimulai. Kamu menyadari banyak jalan yang bisa kamu tempuh.
+Jalan mana yang akan kamu ambil? Hanya kamu yang bisa menentukan.
+
+
+Kamu bosan di rumah, sehingga kamu memutuskan untuk [jalan - jalan](jalan%20-%20jalan/jalan.md).
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.